reference/docs-conceptual/windows-powershell/install/Windows-PowerShell-System-Requirements.md

```diff
@@ -164,28 +143,49 @@ fulfilled by Microsoft .NET Framework 4 and later releases of Microsoft .NET Fra
 
 For information about adding or installing the Windows PowerShell 2.0 engine, and adding or
 installing the required versions of the Microsoft .NET Framework, see
-[Installing the Windows PowerShell 2.0 Engine](Installing-the-Windows-PowerShell-2.0-Engine.md). For
+[Installing the Windows PowerShell 2.0 Engine][23]. For
 information about starting the Windows PowerShell 2.0 engine, see
-[Starting the Windows PowerShell 2.0 Engine](../Starting-the-Windows-PowerShell-2.0-Engine.md).
+[Starting the Windows PowerShell 2.0 Engine][01].
 
 ## Windows Preinstallation Environment
 
 Windows PowerShell 2.0, Windows PowerShell 3.0, and Windows PowerShell 4.0 run in the Windows
 Preinstallation Environment (Windows PE). However, the following cmdlets aren't supported.
```

Comment:
Aside: Does this imply that Windows PowerShell 5.1 doesn't run in Windows PE?

Reply:
https://learn.microsoft.com/windows-hardware/manufacture/desktop/winpe-adding-powershell-support-to-windows-pe